test/unit/sql/index.test.js

```diff
@@ -99,7 +100,7 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
       });
     });
 
-    if (current.dialect.supports.index.using === 2) {
+    if (current.dialect.name !== 'yugabytedb' && current.dialect.supports.index.using === 2) { // yugabytedb uses ybgin in place of gin
```

Comment:
You can resolve that by changing the line

```
using: 'gin',
```

with 

```js
using: current.dialect.name === 'yugabytedb' ? 'ybgin' : 'gin',
```

instead of disabling it, so it's still tested :)